backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/empties/impl/HUEmptiesService.java

```diff
@@ -108,10 +109,16 @@ public void generateMovementFromEmptiesInout(@NonNull final I_M_InOut emptiesInO
 						line.getMovementQty().intValueExact()))
 				.collect(GuavaCollectors.toImmutableList());
 
+		// TODO 1/3 verify that we need to know this
+		final boolean emptiesIncoming = !MovementType.ofCode(emptiesInOut.getMovementType()).isOutboundTransaction();
```

Comment:
loud thinking:

instead of using 
```java
!MovementType.ofCode(emptiesInOut.getMovementType()).isOutboundTransaction()`
```
in a lot of places, maybe it would be better to introduce
```
EmptiesMovementDirection.ofMovementType(String movementType)
```
and use only that everywhere.

You might consider introducing some EmptiesMovementDirection.is* methods if needed.